test/src/module/command/infoTest.c

```diff
@@ -180,6 +183,78 @@ testRun(void)
             "        wal archive min/max (9.4-3): none present\n",
             "text - single stanza, no valid backups");
 
+        // repeat those tests while a backup lock has been taken
+        HARNESS_FORK_BEGIN()
+        {
```

Comment:
Hmmm, interesting, I didn't think these HARNESS_FORK() blocks would be required but now I can see that they are. I'm not very excited about the 1000ms sleeps, but that's not your problem to fix since we do it in other places.

Anyway, I'd like to make this pattern better, but I'm glad you were able to find examples and make it work. Nicely done.

Reply:
Thanks.
Indeed, the "asynchronous" fork need to wait before the lockRelease. It could be really cool to have both fork and parent being able to communicate to wait each-other. But indeed, that's a bigger problem :-)